dice.js

```diff
@@ -0,0 +1,8 @@
+
+function rollDie(){
+    let rand1 = Math.round(Math.random() * 6);
```

Comment:
`let rand1 = Math.round(Math.random() * 6);`
should be `let rand1 = Math.floor(Math.random() * 6)+1;`